src/web/components/Button/Button.test.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import Button from './';
+import { ButtonBase } from './';
```

Comment:
The fileName is always `Button.test` and not `ButtonBase.spec` 😛

Reply:
Sorry :pray: